www/docs/API/account.md

```diff
@@ -96,12 +96,23 @@ account.**declare**(payload [ , transactionsDetail ]) => _Promise < DeclareContr
 
 The _transactionsDetail_ object may include any of:
 
-- transactionsDetail.**maxFee** - Max Fee that that will be used to execute the call(s)
-- transactionsDetail.**nonce** - Nonce for the transaction
-- transactionsDetail.**version** - Version for the transaction (default is 1)
+- transactionsDetail.**contract** - The compiled contract
+- transactionsDetail.**classHash** - Hash of the compiled contract
```

Comment:
- payload.**contract** - The compiled contract
- payload.**classHash** - Hash of the compiled contract

- transactionsDetail.**maxFee** - Max Fee that that will be used to execute the call(s)
- transactionsDetail.**nonce** - Nonce for the transaction
- transactionsDetail.**version** - Version for the transaction (default is 1)

Reply:
done!